flatcar-postinst

```diff
@@ -96,8 +96,9 @@ sysext_download() {
       url="https://bincache.flatcar-linux.net/images/${FLATCAR_BOARD/-usr}/${NEXT_VERSION}/${name}"
     else
       base=$(grep -m 1 -o 'codebase="[^"]*"' "${from}" | cut -d '"' -f 2)
-      entries=$(grep -m 1 -o "<package name=\"${name}\"[^>]*" "${from}")
-      url="${base}/${name}"
+      entries=$(grep -m 1 -o "<package name=\"[^\"]*${name}\"[^>]*" "${from}")
```

Comment:
That doesn't seem to work yet when `name=oem-qemu.gz` and the XML has an entry like `<package name="extrafile-amd64-3815.0.0-oem-qemu.gz" hash="…`

Reply:
```suggestion
      entries=$(grep -m 1 -o "<package name=\"[^\"]*.*${name}\"[^>]*" "${from}")
```
Something like this?

Reply:
I guess I didn't copy correctly, seems to be ok